mods/voidmulti.sc2mod/base.sc2data/gamedata/sounddata.xml

```diff
@@ -270,6 +270,12 @@
         <VolumeRolloffPoints Distance="10.000000"/>
         <VolumeRolloffPoints Distance="20.000000" Volume="-96.000000"/>
     </CSound>
+    <CSound id="Interceptor_AttackImpact">
+        <Volume value="-9.000000,-8.000000"/>
```

Comment:
**undocumented**: Interceptor attack impact sounds are slightly louder (Volume min,max increased from -11,-10 to -9,-8)